File: sedmica 3/FIT_Api_Examples/FIT_Api_Example/Endpoints/Student/GetAll/StudentGetAllResponse.cs
namespace FIT_Api_Example.Endpoints.Student.GetAll
{
    public class StudentGetAllResponse
    {
        public List<StudentGetAllResponseStudent> Studenti { get; set; }
    }

    public class StudentGetAllResponseStudent
    {
        public int ID{ get; set; }
        public string Ime{ get; set; }
        public string Prezime{ get; set; }
        public string OpstinaRodjenjaNaziv{ get; set; }
        public string OpstinaRodjenjaDrzava{ get; set; }
        public DateTime DatumRodjenja{ get; set; }
        public string KorisnickoIme{ get; set; }
        public string SlikaKorisnika { get; set; }
    }
}

File: sedmica 3/front-end/src/app/sedmica5/student-getall-response.ts
export interface StudentGetAllResponse{
  ime: string
  prezime: string
  brojIndeksa: string
  opstinaRodjenjaID: number
  opstinaRodjenja: StudentGetAllResponseOpstinaRodjenja
  datumRodjenja: string
  id: number
  korisnickoIme: string
  slikaKorisnika: string
  isNastavnik: boolean
  isStudent: boolean
  isAdmin: boolean
  isProdekan: boolean
  isDekan: boolean
  isStudentskaSluzba: boolean
}

export interface StudentGetAllResponseOpstinaRodjenja{
  id: number
  description: string
  drzavaID: number
  drzava: StudentGetAllResponseDrzava
}


export interface StudentGetAllResponseDrzava {
  id: number
  naziv: string
  skracenica: any
}
